### ISSUE TITLE
Missing explanation of Errata directory

### ISSUE THREAD

**Issue by Jack-Etheredge** [DeepReinforcementLearning/DeepReinforcementLearningInAction#33]
Is the Errata folder in the github repo the correct versions of the notebooks or older incorrect versions? I think it would be helpful to have that information in the README.md or to simply include only corrected versions of the notebooks.

**Comment by outlace** [comment]
Good point - I've updated readme. yes errata folder notebooks is more up-to-date in terms of catching the errata

**Comment by Jack-Etheredge** [comment]
Excellent. Thank you.
